src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -237,34 +237,6 @@ def cancelWorkflow(self, wf):
         elements = [x['id'] for x in data.get('rows', []) if x['key'][1] not in nonCancelableElements]
         return self.updateElements(*elements, Status='CancelRequested')
 
-    def updateSiteLists(self, wf, siteWhiteList=None, siteBlackList=None):
```

Comment:
Are there any unit tests for this method? If so, please also delete it/them.

Reply:
I already checked , there is no unit test for this method.